hashicorp/boundary-ui
# refactor: 💡 remove route-action usage for workers

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12613

# Description

Remove route-action usage for workers. The only route-action usage that was not removed was the actions used for tag filtering because that will be replaced by worker tag ux improvements in the future.




## Screenshots (if appropriate)

## How to Test

1. start boundary dev instance
2. test editing and deleting an existing worker
3. start boundary server instance in order to get a registration token
4. use token to create a worker (make sure cancel and done buttons behave as intended in pki worker creation form)

## Checklist
